pkg/agent/openflow/network_policy.go

```diff
@@ -197,7 +197,7 @@ func (ctx *conjMatchFlowContext) installOrUpdateFlow(actions []*conjunctiveActio
 		// Build the Openflow entry. actions here should not be empty for either add or update case.
 		flow := ctx.client.conjunctiveMatchFlow(ctx.tableID, ctx.matchKey, ctx.matchValue, actions...)
 
-		if err := flow.Add(); err != nil {
+		if err := ctx.client.flowOperations.Add(flow); err != nil {
```

Comment:
I assume you want to change NetworkPolicy to use bundles in a separate PR later?

Reply:
Makes sense to me.